src/resources/components/app/main-content.tsx

```diff
@@ -0,0 +1,32 @@
+import AboutCard from "@Components/about/about";
+import ChatAppearance from "@Components/settings/chat-appearance";
+import ProfileCard from "@Components/settings/profile-card";
+import SettingsForm from "@Components/settings/settings-form";
+import type UserStorageService from "~services/user/user-storage-service";
+import Stats from "../stats/stats";
+import Theme from "../theme/theme";
+
+type MainContentProps = {
+  selectedItem: string;
+  userService: UserStorageService;
+};
+
+const MainContent = ({ selectedItem, userService }: MainContentProps) => {
+  switch (selectedItem) {
+    case "Configurações":
```

Comment:
Cases with non-ascii characters isn't the best of the ideas. Any reason for it being this way?

Reply:
It was the quickest way to implement the feature, but I'm going to switch the keys to English.